jujubundlelib/tests/test_pyutils.py

```diff
@@ -17,7 +17,7 @@ def test_python2(self):
         byte_string = str(instance)
         self.assertIsInstance(byte_string, bytes)
         self.assertEqual(b'example', byte_string)
-        unicode_string = unicode(instance)
+        unicode_string = unicode(instance)  # noqa: F821
```

Comment:
unicode is no longer present in Py3, is it?

Reply:
here, `unicode` in py3 takes the bytestr to a str, and in py2 takes a str to unicode; so that the test has basically the same behaviour.

Reply:
Cool.